app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -884,18 +883,13 @@ export class AppCommandParser {
             return [];
         }
 
-        const values: AppLookupCallValues = {
-            name: f.name,
-            user_input: parsed.incomplete,
-            values: parsed.values,
-        };
-
         const payload = this.composeCallFromParsed(parsed);
         if (!payload) {
             return [];
         }
         payload.type = AppCallTypes.LOOKUP;
-        payload.values = values;
+        payload.selected_field = f.name;
+        payload.query = parsed.incomplete;
```

Comment:
Are you missing here payload.values? Or is this assigned somewhere different?

Reply:
@larkox They are assigned in `composeCallFromParsed`. I can put assign them again here for verbosity if you think it should be